### PR TITLE
Plans: Tweak Jetpack plans grid wording

### LINKED ISSUE #22701: Plans: Incorrect tooltip copy
#### Steps to reproduce
1. Starting at URL: https://wordpress.com/plans/{site-url}
2. Click popover for "Unlimited storage space" under personal plan
3. Click popover for "Monetize your site" under premium plan

#### What I expected
Correct description.

#### What happened instead
1. Storage space should be talking about backups
2. Monetize should also be talking about PayPal payments

#### Screenshot / Video

richard_s_sandbox_ _wordpress_com

richard_s_sandbox_ _wordpress_com

### PR DESCRIPTION
**Pull request by ebinnion**
Fixes #22701

This PR fixes a couple of specific plans tooltip issues that @richardmuscat reported and also makes several other copy edits that @benhuberman suggested in #22701.

Here's a screenshot of the Jetpack plans grid after this PR:

screen shot 2018-03-07 at 11 34 20 am

From what I can tell, this change only affects the monetization feature for WP.com plans, so be sure to check that out on a WP.com site.

Many of these copy edits can also be applied to WordPress.com plans. I'll follow-up with a PR for WordPress.com after this one goes in.

Here are some other screenshots of tooltips that were changed:

screen shot 2018-03-07 at 11 40 49 am
screen shot 2018-03-07 at 11 41 26 am
screen shot 2018-03-07 at 11 41 05 am
screen shot 2018-03-07 at 11 41 10 am
screen shot 2018-03-07 at 11 41 14 am
screen shot 2018-03-07 at 11 41 20 am
screen shot 2018-03-07 at 11 40 54 am
screen shot 2018-03-07 at 11 40 58 am

### CODE REVIEW

File: client/lib/plans/constants.js [hunk 1]
```diff
@@ -1408,10 +1399,10 @@ export const FEATURES_LIST = {
 
 	[ FEATURE_SEARCH ]: {
 		getSlug: () => FEATURE_SEARCH,
-		getTitle: () => i18n.translate( 'Enhanced site-wide search' ),
+		getTitle: () => i18n.translate( 'Enhanced Site-wide Search' ),
 		getDescription: () =>
 			i18n.translate(
-				'Fast, high-quality search results with custom filtering, powered by Elasticsearch.'
+				'Fast, relevant search results with custom filtering, powered by Elasticsearch.'
```
Comment (a8ci18n) [inline client/lib/plans/constants.js]: :ok: This change is safe, we'll keep the existing 17 translations.

File: client/lib/plans/constants.js [hunk 2]
```diff
@@ -1258,7 +1249,7 @@ export const FEATURES_LIST = {
 		getTitle: () => i18n.translate( 'Daily Malware Scanning' ),
 		getDescription: () =>
 			i18n.translate(
-				'Comprehensive and automated scanning for any security vulnerabilities or threats on your site.'
+				'Comprehensive, automated scanning for security vulnerabilities or threats on your site.'
```
Comment (a8ci18n) [inline client/lib/plans/constants.js]: :ok: This change is safe, we'll keep the existing 19 translations.

File: client/lib/plans/constants.js [hunk 3]
```diff
@@ -1192,15 +1184,14 @@ export const FEATURES_LIST = {
 	[ FEATURE_SPAM_AKISMET_PLUS ]: {
 		getSlug: () => FEATURE_SPAM_AKISMET_PLUS,
 		getTitle: () => i18n.translate( 'Spam Protection' ),
-		getDescription: () => i18n.translate( 'State-of-the-art spam defense powered by Akismet.' ),
+		getDescription: () => i18n.translate( 'State-of-the-art spam defense, powered by Akismet.' ),
```
Comment (a8ci18n) [inline client/lib/plans/constants.js]: :ok: This change is safe, we'll keep the existing 23 translations.

File: client/lib/plans/constants.js [hunk 4]
```diff
@@ -1075,10 +1070,10 @@ export const FEATURES_LIST = {
 
 	[ FEATURE_WORDADS_INSTANT ]: {
 		getSlug: () => FEATURE_WORDADS_INSTANT,
-		getTitle: () => i18n.translate( 'Monetize Your Site' ),
+		getTitle: () => i18n.translate( 'Monetization Options' ),
```
Comment (a8ci18n) [inline client/lib/plans/constants.js]: Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Easy monetization options' )` *ES Score: 21*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/lib/plans/constants.js [hunk 5]
```diff
@@ -561,7 +560,7 @@ export const PLANS_LIST = {
 		getPathSlug: () => 'jetpack-personal',
 		getDescription: () =>
 			i18n.translate(
-				'Security essentials for your WordPress site including ' +
+				'Security essentials for your WordPress site, including ' +
```
Comment (a8ci18n) [inline client/lib/plans/constants.js]: Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Security essentials for your WordPress site including automated backups and priority support.' )` *ES Score: 18*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/22701-features-tweaks)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by AnnaMag** [comment]
repeated efforts? -> https://github.com/Automattic/wp-calypso/pull/23059

**Comment by joanrho** [comment]
@ebinnion I've compared this to #23059 and added some edits you had that my PR did not. Thanks for this. Closing.

**Comment by joanrho** [comment]
@benhuberman no worries, the period is there in the PR I merged yesterday. Thanks for catching!

**Pull request closed without merging**